src/test/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesSchedulerIT.java

```diff
@@ -526,6 +528,46 @@ public void testJobAnnotationsFromSchedulerProperties() {
 		CronJob cronJob = kubernetesScheduler.createCronJob(scheduleRequest);
 
 		assertThat(cronJob.getMetadata().getAnnotations().get("test1")).as("Job annotation is not set").isEqualTo("value1");
+		assertThat(cronJob.getSpec().getJobTemplate().getSpec().getTemplate().getMetadata().getAnnotations().get("podtest1")).as("Pod annotation is not set").isEqualTo("podvalue1");
```

Comment:
I think it would be a good idea to have a empty label test much like the original.   Where it checks the following:
```
		assertThat(cronJob.getSpec().getJobTemplate().getSpec().getTemplate().getMetadata().getAnnotations()).isNull();
		assertThat(cronJob.getSpec().getJobTemplate().getSpec().getTemplate().getMetadata().getLabels().size()).as("Should have one label").isEqualTo(1);
```

Reply:
Add a test with this suggestion. Thanks!